x-pack/plugins/security_solution/public/common/components/ml_popover/hooks/use_security_jobs.ts

```diff
@@ -35,25 +38,24 @@ export interface UseSecurityJobsReturn {
  * NOTE: If the user is not an ml admin, jobs will be empty and isMlAdmin will be false.
  * If you only need installed jobs, try the {@link useInstalledSecurityJobs} hook.
  *
- * @param refetchData
  */
-export const useSecurityJobs = (refetchData: boolean): UseSecurityJobsReturn => {
+export const useSecurityJobs = (): UseSecurityJobsReturn => {
   const [jobs, setJobs] = useState<SecurityJob[]>([]);
   const [loading, setLoading] = useState(true);
   const mlCapabilities = useMlCapabilities();
   const [securitySolutionDefaultIndex] = useUiSetting$<string[]>(DEFAULT_INDEX_KEY);
   const http = useHttp();
   const { addError } = useAppToasts();
-
+  const refetch = useRef<inputsModel.Refetch>(noop);
```

Comment:
For example, refetching is natively supported by React Query and is a matter of simple config for a given hook.